test/rewrite_clj/parser_test.clj

```diff
@@ -132,6 +132,17 @@
   "\"123\n456\""  :multi-line  "123\n456")
 
 (tabular
+ (fact "about parsing regex-strings"
```

Comment:
Also noticed there's an existing fact for this, I'll just update the existing one.